src/components/gb-card-4 - Copy.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
```

Comment:
don't mind the comments down there, please delete this gb-card-4 file!

Reply:
Done

Reply:
done!